extension/src/experiments/index.ts

```diff
@@ -602,4 +609,19 @@ export class Experiments extends BaseRepository<TableData> {
       this.onDidChangeColumns
     )
   }
+
+  private async checkSignalFile() {
+    const dvcLiveOnly = await checkSignalFile(this.dvcLiveOnlySignalFile)
+
+    if (dvcLiveOnly && !this.dvcLiveOnlyCleanupInitialized) {
+      this.dvcLiveOnlyCleanupInitialized = true
+      pollSignalFileForProcess(this.dvcLiveOnlySignalFile, () => {
+        this.dvcLiveOnlyCleanupInitialized = false
+        if (this.hasRunningExperiment()) {
+          this.cliData.update()
```

Comment:
[F] The signal file has now been removed so we request the data again to flush out the fact that an experiment is no longer running.